model/page.go

```diff
@@ -1,35 +1,40 @@
-	HasJSONLD                        bool            `json:"has_jsonld"`
-	FeatureFlags                     FeatureFlags    `json:"feature_flags"`
-	Error                            Error           `json:"error"`
-	EmergencyBanner                  EmergencyBanner `json:"emergency_banner"`
-	Collapsible                      Collapsible     `json:"collapsible"`
-	Pagination                       Pagination      `json:"pagination"`
-	TableOfContents                  TableOfContents `json:"table_of_contents"`
-	BackTo                           BackTo          `json:"back_to"`
-	SearchNoIndexEnabled             bool            `json:"search_no_index_enabled"`
+	Count                            int                    `json:"count"`
+	Type                             string                 `json:"type"`
+	DatasetId                        string                 `json:"dataset_id"`
+	DatasetTitle                     string                 `json:"dataset_title"`
+	URI                              string                 `json:"uri"`
+	Taxonomy                         []TaxonomyNode         `json:"taxonomy"`
+	Breadcrumb                       []TaxonomyNode         `json:"breadcrumb"`
+	IsInFilterBreadcrumb             bool                   `json:"is_in_filter_breadcrumb"`
+	ServiceMessage                   string                 `json:"service_message"`
+	Metadata                         Metadata               `json:"metadata"`
+	SearchDisabled                   bool                   `json:"search_disabled"`
+	SiteDomain                       string                 `json:"-"`
+	PatternLibraryAssetsPath         string                 `json:"-"`
+	Language                         string                 `json:"language"`
+	IncludeAssetsIntegrityAttributes bool                   `json:"-"`
+	ReleaseDate                      string                 `json:"release_date"`
+	BetaBannerEnabled                bool                   `json:"beta_banner_enabled"`
+	CookiesPreferencesSet            bool                   `json:"cookies_preferences_set"`
+	CookiesPolicy                    CookiesPolicy          `json:"cookies_policy"`
+	HasJSONLD                        bool                   `json:"has_jsonld"`
+	FeatureFlags                     FeatureFlags           `json:"feature_flags"`
+	Error                            Error                  `json:"error"`
+	EmergencyBanner                  EmergencyBanner        `json:"emergency_banner"`
+	Collapsible                      Collapsible            `json:"collapsible"`
+	Pagination                       Pagination             `json:"pagination"`
+	TableOfContents                  TableOfContents        `json:"table_of_contents"`
+	BackTo                           BackTo                 `json:"back_to"`
+	SearchNoIndexEnabled             bool                   `json:"search_no_index_enabled"`
+	NavigationContent                *topicModel.Navigation `json:"navigation_content"`
```

Comment:
If you specify the banner data instead of using topicModel, the dependency issue will go away. Something like the following:

```suggestion
	NavigationContent                Navigation `json:"navigation_content"`
}

type Navigation struct {
	Items       *[]Topic `json:"items,omitempty"`
}

type Topic struct {
	Label         string                 `json:"label"`
	SubtopicItems *[]Topic               `json:"subtopics,omitempty"`
	Uri           string                 `json:"uri"`
}
```

Reply:
Fyi - I did a quick check of the fields used in partial template, so in the above extend for only fields you need for navbar